lib/rest/Endpoints.js

```diff
@@ -55,6 +56,7 @@ module.exports.GUILD_WEBHOOKS =                                        (guildID)
 module.exports.GUILD_WIDGET =                                          (guildID) => `/guilds/${guildID}/widget`;
 module.exports.GUILDS =                                                             "/guilds";
 module.exports.INVITE =                                               (inviteID) => `/invite/${inviteID}`;
+module.exports.MESSAGE =                         (guildID, channelID, messageID) => `/channels/${guildID}/${channelID}/${messageID}`;
```

Comment:
This isn't an endpoint we're sending a request to, so this should probably be defined elsewhere. Should either be at the bottom of the file, or otherwise specified directly in the Message class.

Reply:
You think it would be ok if I just hardcoded this URL into the getter?